sharktank/sharktank/layers/sharding.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2024 Advanced Micro Devices, Inc.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+"""Specifications describing how layers are sharded."""
+
+from ..types.sharding import *
+
+
+class PagedLlamaAttentionBlockSharding(ThetaLayerSharding):
```

Comment:
This should stay in `models` as its more of a model specific sharding.

Reply:
I moved it back.